app/knights/apply_armour.py

```diff
@@ -0,0 +1,8 @@
+class Armour:
+
+    @staticmethod
+    def app_armour(armour: dict):
+        protection = 0
+        for a in armour:
```

Comment:
`a` is an uninformative name for variable

Reply:
return self.hidden is True if self.hidden is False else self.hidden is False
When I write in one line  - not pass all tests

Reply:
That comment shouldn't have been there